src/lib/convenience/RegisterProviderRequest.h

```diff
@@ -50,7 +50,7 @@ typedef struct RegisterProviderRequest
   RegisterProviderRequest();
 
   std::string  render(Format format, std::string indent);
-  std::string  check(RequestType requestType, Format format, std::string indent, std::string preError, int counter);
+  std::string  check(ConnectionInfo* ciP, RequestType requestType, Format format, std::string indent, std::string preError, int counter);
```

Comment:
@kzangeli mentioned an old issue about aligned the check() signatures. I'd be great to know the issue number, as probably this PR is closing it.

@kzangeli did you remember?